composer.json

```diff
@@ -28,8 +28,8 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.6.4",
-        "laravel/framework": "5.3.* || 5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.* || 6.* || 7.*",
-        "guzzlehttp/guzzle": "6.*"
+        "laravel/framework": "5.3.* || 5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.* || 6.* || 7.* || 8.*",
+        "guzzlehttp/guzzle": "7.*"
```

Comment:
I think it should be "6.* || 7.*" to support older versions of laravel and guzzle

Reply:
@mostafaznv you're right. I've updated the pull request to reflect the change 👍